setup.py

```diff
@@ -18,24 +21,55 @@
     profile = True
     del sys.argv[sys.argv.index("--profile")]
 
-compilation_includes = [".", numpy.get_include()]
+if "--line-profile" in sys.argv:
+    line_profile = True
+    del sys.argv[sys.argv.index("--line-profile")]
 
+if "--install-rates" in sys.argv:
+    install_rates = True
+    del sys.argv[sys.argv.index("--install-rates")]
```

Comment:
Likewise, I think this can be removed for cherab/experimental.

Reply:
removed